typescript/src/shared/parameters.ts

```diff
@@ -113,3 +113,15 @@ export const createRefundParameters = z.object({
     .optional()
     .describe('The amount to refund in cents.'),
 });
+
+export const searchDocumentationParameters = z.object({
+  question: z
+    .string()
+    .describe('The question to ask about Stripe documentation.'),
+  language: z
+    .string()
```

Comment:
should be an enum

```
[
        "dotnet",
        "go",
        "java",
        "node",
        "php",
        "ruby",
        "python",
        "curl"
      ]
```

Reply:
(from here: https://stripe.sourcegraphcloud.com/stripe-internal/pay-server/-/blob/developer_ai/command/chat/query_code_snippets.rb?L26-35)